nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/AbstractPutHBase.java

```diff
@@ -221,9 +222,11 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             try {
                 clientService.put(entry.getKey(), entry.getValue());
                 successes.addAll(entry.getValue());
-            } catch (Exception e) {
+            } catch (final KerberosLoginException kle) {
+                getLogger().error("Failed to connect to HBase due to {}. Rolling back session, and penalizing flow files.", new Object[]{kle}, kle);
```

Comment:
The wrapping `Object[]` can be removed:
```suggestion
                getLogger().error("Failed to connect to HBase due to {}: Rolling back session, and penalizing flow files", kle, kle);
```